nemo/backends/pytorch/actions.py

```diff
@@ -946,17 +953,17 @@ def __extract_dynamic_axes(port_name: str, ntype: NeuralType, dynamic_axes: defa
         if len(dynamic_axes) == 0:
             dynamic_axes = None
 
-        local_parameters = {}
```

Comment:
can you please revert this part back to using "local_parameters" since it is kept (but deprecated)?
We will change this in a separate PR, most likely, to a different logic anyway.

Reply:
not sure what do you mean? please note that in actual file I am saving:
                with open(output + ".json", 'w') as outfile:
                    json.dump(init_params_copy, outfile)
so you want me to... revert the variable that won't be used?